targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/device/TOOLCHAIN_ARM_STD/startup_stm32f429xx.S

```diff
@@ -36,8 +36,11 @@
 ;* OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ; 
 ;*******************************************************************************
-
+#if !defined(__MICROLIB)
```

Comment:
As discussed in confluence, this is probably not required. Try without it

Reply:
yes removed and kept one initial_sp config